composer.json

```diff
@@ -27,7 +27,7 @@
     },
     "require-dev": {
         "psr/log": "~1.0",
-        "phpcr/phpcr-api-tests": "2.1.8",
+        "phpcr/phpcr-api-tests": "dev-escape-property-name as 2.1.9",
```

Comment:
adjust when https://github.com/phpcr/phpcr-api-tests/pull/171 is merged and tagged